src/ape/api/networks.py

```diff
@@ -326,22 +338,16 @@ class NetworkAPI:
     ecosystem: EcosystemAPI
     """The ecosystem of the network."""
 
-    config_manager: "ConfigManager"
-    """A reference to the global config manager."""
-
-    plugin_manager: PluginManager
-    """A reference to the global plugin manager."""
-
     data_folder: Path  # For caching any data that might need caching
     """The path to the ``.ape`` directory."""
 
-    request_header: str
+    request_header: dict
     """A shareable network HTTP header."""
 
-    _default_provider: str = ""
+    _default_provider: Optional[str] = ""
```

Comment:
Is this supposed to be `Optional`? I would think it would default to `None` rather than the `""` if that is the case.